02.create-and-explore/05.avatars/02.create-avatars/docs.md

```diff
@@ -97,8 +97,9 @@ Now you can package your avatar to use it in High Fidelity.
 
 1. Launch High Fidelity’s Interface application.
 2. Enable ‘Advanced Menus’ under Settings > Advanced Menus.
```

Comment:
I think this step is not necessary since "Advanced Menus" option has been removed since RC68